modules/ossm-release-3-0-TP-1.adoc

```diff
@@ -9,10 +9,57 @@ Module included in the following assemblies:
 
 This release of {SMProductName} is Technology Preview.
 
-// Simple table to help clarify for users the component versions they can use with OSSM 3.0 TP1. Likely a more complicated table will be needed for GA. OSSM 3.0 separates all the Operators such as OCP, Istio, Envoy, Kiali, OTEL/Tempo, etc. This is only what is known to work with OSSM 3.0 TP1.
+// Simple table to help clarify for users the component versions they can use with OSSM 3.0 TP1. Likely a more complicated table will be needed for GA. OSSM 3.0 separates all the Operators such as OCP, Istio, Envoy, Kiali, OTEL/Tempo, etc. This is only what is known to work with OSSM 3.0 TP1 and TP2, as of 12/09/2024.
+
+[id="ossm-release-3-0-TP1-component-versions-2_{context}"]
+== Supported component versions for Technology Preview 2
+
+|===
+|Component |Version
+
+|{ocp-product-title}
+|4.14+
+
+|Istio
+|1.24.1
+
+|Envoy Proxy
+|1.32
+
+|Kiali Operator
+|2.1
+
+|Kiali Server
+|2.1
+|===
+
+[IMPORTANT]
+====
+* You need to remove the Kiali custom resources (CR) from Technology Preview 1 before you update to {KialiProduct} 2.1.
+
+* {KialiProduct} 2.1 is available in the `candidates` channel.
++
+The `candidate` channel offers unsupported early access to releases as soon as they are built. Releases present only in candidate channels may not contain the full feature set of eventual GA releases or features may be removed prior to GA. Additionally, these releases have not been subject to full Red Hat Quality Assurance and may not offer update paths to later GA releases. Given these caveats, the candidate channel is only suitable for testing purposes where destroying and recreating a cluster is acceptable.
+====
+//copied from https://docs.openshift.com/container-platform/4.17/updating/understanding_updates/understanding-update-channels-release.html#candidate-version-channel_understanding-update-channels-releases but per engineering, it is "candidates" channel, not "candidates-"
+
+[id="ossm-release-3-0-TP1-unavailable-features-2_{context}"]
+== Unavailable features in Technology Preview 2
+
+The following features are not supported in the Technology Preview 2 release:
+
+* Istio's Ambient mode
```

Comment:
From [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=references-product-service-names#general-guidelines): Do not use ’s to show the possessive form of brand names or product names.
```suggestion
* Ambient mode in Istio
```